tests/src/whisk/core/cli/test/WskBasicTests.scala

```diff
@@ -38,24 +40,89 @@ import common.TestUtils.SUCCESS_EXIT
 import common.TestUtils.TIMEOUT
 import common.TestUtils.UNAUTHORIZED
 import common.Wsk
+import common.WskAdmin
 import common.WskProps
 import common.WskTestHelpers
 import spray.json._
 import spray.json.DefaultJsonProtocol._
 import spray.json.JsObject
 import spray.json.pimpAny
 import spray.json.pimpString
-import whisk.core.entity.WhiskPackage
+import whisk.core.entity.AuthKey
 import whisk.core.entity.size.SizeInt
+import whisk.core.entity.Subject
+import whisk.core.entity.WhiskAuth
+import whisk.core.entity.WhiskPackage
 
 @RunWith(classOf[JUnitRunner])
 class WskBasicTests
     extends TestHelpers
-    with WskTestHelpers {
+    with WskTestHelpers
+    with BeforeAndAfterAll {
 
     implicit val wskprops = WskProps()
     val wsk = new Wsk(usePythonCLI = false)
-    val defaultAction = Some(TestUtils.getCatalogFilename("samples/hello.js"))
+    val defaultAction = Some(TestUtils.getTestActionFilename("hello.js"))
+    val samplePackage = "samplePackage"
+    val sampleAction = samplePackage + "/sampleAction"
+    val sampleFeed = samplePackage + "/sampleFeed"
+    val wskadmin = new RunWskAdminCmd {}
+
+    var authkey = ""
+    var namespace = ""
+    var fullSampleActionName = ""
+    var fullSampleFeedName = ""
+    var fullSamplePackageName = ""
+
+    override def beforeAll() {
+        var auth = WhiskAuth(Subject(), AuthKey())
```

Comment:
a clue is that you're not using `auth` anywhere other than `auth.subject` which you can accomplish with just `Subject()`.